tests/testthat/test-module_teal.R

```diff
@@ -645,7 +825,7 @@ testthat::describe("srv_teal teal_modules", {
             trimws(
               rvest::html_text2(
                 rvest::read_html(
-                  output[["validate-shiny_warnings-message"]]$html
+                  output[["teal-datanames_warning-message"]]$html
```

Comment:
I am having hard time fixing this test.

The `selector` inside this app

```r
app <- teal::init(
  data = teal_data(mtcars = mtcars),
  modules = modules(
  module("module_1", datanames = c("missing1", "missing2"))
  )
)

shinyApp(app$ui, app$server)

```

is

```r
#teal-datanames_warning-message > div > span:nth-child(1) > span:nth-child(5)
```

![image](https://github.com/user-attachments/assets/81173a55-52e1-4d0c-af7b-7c4cc8f97044)

Reply:
Solved in 0860bdf94

Reply:
Thanks, I thought I tried that as well locally : P